msg/VehicleCommandAck.msg

```diff
@@ -25,7 +25,7 @@ uint8 ORB_QUEUE_LENGTH = 4
 
 uint32 command						# Command that is being acknowledged
 uint8 result						# Command result
-uint8 result_param1					# Also used as progress[%], it can be set with the reason why the command was denied, or the progress percentage when result is MAV_RESULT_IN_PROGRESS
+uint8 progress					# Also used as progress[%], it can be set with the reason why the command was denied, or the progress percentage when result is MAV_RESULT_IN_PROGRESS
```

Comment:
Perhaps change comment so that it is used as progress, and that it can also be used for other purposes?